app/src/androidTest/java/com/github/sdpcoachme/profile/EditTextActivityTest.kt

```diff
@@ -0,0 +1,41 @@
+package com.github.sdpcoachme.profile
+
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createEmptyComposeRule
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+import com.github.sdpcoachme.profile.EditTextActivity.TestTags.Companion.TITLE
+import com.github.sdpcoachme.profile.EditTextActivity.TestTags.Companion.TextFields.Companion.MAIN
+import org.junit.Rule
+import org.junit.Test
+
+class EditTextActivityTest {
+
+    @get:Rule
+    val composeTestRule = createEmptyComposeRule()
+
+    @Test
+    fun intentParametersAreDisplayedProperly() {
+        val title = "Title"
+        val placeholder = "Placeholder"
+        val label = "Label"
+        val initialValue = "Initial value"
+        val intent = EditTextActivity.getIntent(
+            context = ApplicationProvider.getApplicationContext(),
+            title = title,
+            placeholder = placeholder,
+            label = label,
+            initialValue = initialValue
+        )
+        ActivityScenario.launch<EditTextActivity>(intent).use {
+            composeTestRule.onNodeWithTag(MAIN).assert(hasText(initialValue))
+            composeTestRule.onNodeWithTag(TITLE).assertTextEquals(title)
+            composeTestRule.onNodeWithTag(MAIN).performTextClearance()
+            composeTestRule.onNodeWithTag(MAIN).performClick()
+            composeTestRule.onNodeWithText(placeholder).assertIsDisplayed()
+            composeTestRule.onNodeWithText(label).assertIsDisplayed()
+        }
+    }
+
```

Comment:
Nice job with the edit text activity. This will make the user experience a lot nicer!

Just a little idea for an additional test:
I believe that the default values are not tested for this activity (e.g., where initialValue is set to "", title is set to DEFAULT_TITLE, etc.). By adding a test for this case, we could increase test coverage a little.